tests/bases/test_collections.py

```diff
@@ -168,6 +168,25 @@ def test_metric_collection_prefix_postfix_args(prefix, postfix):
     for name in names:
         assert f"new_prefix_{name}" in out, 'prefix argument not working as intended with clone method'
 
+    for k, _ in new_metric_collection.items():
+        assert 'new_prefix_' in k
+
+    for k in new_metric_collection.keys():
+        assert 'new_prefix_' in k
+
+    for k, _ in new_metric_collection.items(keep_base=True):
+        assert 'new_prefix_' not in k
+
+    for k in new_metric_collection.keys(keep_base=True):
+        assert 'new_prefix_' not in k
+
+    repr = 'MetricCollection(' \
+           '\n  prefix=new_prefix_,' \
```

Comment:
```suggestion
           f'\n  prefix={prefix},' \
```
why not also this?

Reply:
Because the prefix was fixed. I added a proper test for repr.